### PR TITLE
Update CN docs

### PR DESCRIPTION
**Pull request by JaylinYu**

### CODE REVIEW

**Comment by JaylinYu** [comment]
Seems like I have trouble making tabs right?

File: docs/zh_CN/bridges/quic-bridge.md [hunk 1]
```diff
@@ -2,14 +2,19 @@
 
 NanoMQ 已支持 MQTT over QUIC 桥接，用户可以使用 QUIC 作为 MQTT 协议的传输层来与 EMQX 5.0 消息服务建立桥接进行数据同步，从而为无法集成或找到合适的 MQTT over QUIC SDK 的端侧设备和难以修改固件的嵌入式设备提供在 IoT 场景利用 QUIC 协议优势的捷径。依靠 EMQX+NanoMQ 的云边一体化的消息架构，用户能够快速且低成本的在泛物联网场景中完成跨时空地域的数据采集和同步需求。
 
-支持特性：
+QUIC bridging shares following exclusive features and special advantages
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:5]:
```suggestion

```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 2]
```diff
@@ -2,14 +2,19 @@
...
 
-- 多流传输 
-- 混合桥接模式 
-- 设置 QoS 消息更高的优先传输级别
-- 初始的 RTT （ Round Trip Time ） 预测值设置
-- 重置 QUIC 传输层拥塞控制检测的最大空闲时间
-- TLS 双向认证
+- Multi-stream : Topic-Stream pair, avoid of head of line blocking problem.
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:7]:
```suggestion

```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 3]
```diff
@@ -2,14 +2,19 @@
...
+- Hybird bridging : automatically downgrade to TCP if QUIC is not availiable
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:8]:
```suggestion

```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 4]
```diff
@@ -2,14 +2,19 @@
...
+- Message prioritization: Assign High priority for QoS (1|2) message to ensure bandwidth usage
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:9]:
```suggestion

```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 5]
```diff
@@ -2,14 +2,19 @@
...
+- O-RTT Quick reconnect : 0 RTT（Round Trip Time）estimate time
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:10]:
```suggestion

```

File: docs/zh_CN/bridges/quic-bridge.md [hunk 6]
```diff
@@ -253,3 +284,4 @@ quic_strm_cb: QUIC_STREAM_EVENT_START_COMPLETE [0x618000020080] ID: 4 Status: 0
 ```
 
 之后 NanoMQ 就会自动根据 Topic 将数据包导流至不同的 Stream 发送。经过内部测试，在使用模拟 2s 延迟和 40% 丢包的弱网环境时，多流桥接可以显著降低延时。
+同时，QoS 1/2 消息报文享有相较于 QoS 0 更高的优先级，依靠此区别 NanoMQ 可以帮助用户更好的分配和管理网络带宽的使用，将有限且宝贵的带宽资源留给更有价值的数据，从而避免高价值数据因为网络拥塞而无法同步导致的数据丢失。
```
Comment (LenaLenaPan) [inline docs/zh_CN/bridges/quic-bridge.md:287]:
```suggestion
同时，QoS 1/2 消息报文享有相较于 QoS 0 更高的优先级，用户可基于此更好地分配和管理网络带宽的使用，将有限且宝贵的带宽资源留给更有价值的数据，避免因网络拥塞导致造成的高价值数据的丢失。
```

**Review by LenaLenaPan — COMMENTED** [review]
leave some comments

**Review by LenaLenaPan — APPROVED** [review]

**Pull request merged**